src/helpers/ratingsInsertion.js

```diff
@@ -0,0 +1,41 @@
+import db from '../database/models';
+import BaseRepository from '../repository/base.repository';
```

Comment:
move this logic to your controller.

This is a controller.

Reply:
helpers should be small utility functions that are used across controller methods